app/articles/page.tsx

```diff
@@ -1,26 +1,57 @@
 import Link from "next/link";
 import { notFound } from "next/navigation";
-
+import styles from "./index.module.scss";
 import { httpGet$GetPosts } from "@/modules/commands/GetPosts/fetcher";
+import Navbar from "../_containers/Navbar";
+import Footer from "../_containers/Footer";
+import Posts from "../_containers/Posts";
 
-export default async function Page() {
-  const data = await httpGet$GetPosts(`/posts`, {}).catch((error) => {
-    console.log(error);
-  });
-  if (!data) {
-    notFound();
+
+interface Post {
+  id: number
+  title: string
+  description: string
+  imageUrl: string
+  tags: string[]
+}
+
+const dummyPosts: Post[] = [
```

Comment:
Don't use this..

We already deployed [wordpress](https://wp.ecoquity.club/). For now, we can actually fetch the posts directly from the site, instead of creating the dummy posts.